lib/t/feeds.ex

```diff
@@ -226,6 +234,7 @@ defmodule T.Feeds do
     |> not_liked_profiles_q(user_id)
     |> not_liker_profiles_q(user_id)
     |> not_seen_profiles_q(user_id)
+    |> not_expired_match_profiles_q(user_id)
```

Comment:
```suggestion
    |> not_expired_match_with_q(user_id)
```
or something similar `not_expired_match_profiles_q` doesn't sound right